README.md

```diff
@@ -62,6 +62,16 @@ ni --frozen
 # bun install --frozen-lockfile
 ```
 
+```bash
+ni --frozen -P
```

Comment:
I don't think we need to demonstrate this - this was more like a bug, people should expect this to work.